students/10th/songsuho/js/login.js

```diff
@@ -0,0 +1,13 @@
+let info = document.getElementsByClassName('idInput')[0];
+let pwd = document.getElementsByClassName('pwInput')[0];
+
+console.log(info, pwd)
+document.addEventListener('keyup',function(e) {
+    if (info.value !== "" && pwd.value !== "") { 
+            let color = document.getElementById('loginbtn');
+            color.style.backgroundColor = 'blue';
+        }
+});
```

Comment:
1. 함수 내에서 e(이벤트 객체)가 사용되지 않으니 지워주세요.

지우시기 전에 함수 안에 console.log(e) / console.log(e.target) / console.log(e.target.value) 바꿔가면서 각각의 값이 어떻게 나타나는지 확인해보면 좋겠네요.

2. indent 지켜주세요!

```suggestion
document.addEventListener('keyup', function(e) {
  if (info.value !== "" && pwd.value !== "") { 
    let color = document.getElementById('loginbtn');
    color.style.backgroundColor = 'blue';
   }
})
```

3. color 변수에 담아주신 #loginbtn 요소는 함수가 호출되어 조건문을 통과할 때마다 정의되고 있네요. 동작에는 문제가 없지만 코드 가독성과 효율을 위해 info, pwd와 함께 두어도 좋겠습니다.